src/LibraryManager.Contracts/FileHelpers.cs

```diff
@@ -304,5 +324,28 @@ public static bool DeleteFileFromDisk(string filePath)
                 return false;
             }
         }
+
+        private static bool IsUNCPath(string rootPath)
+        {
+            return new Uri(rootPath).IsUnc;
+        }
+
+        private class LocalPathComparer : IEqualityComparer<string>
+        {
+            public bool Equals(string path1, string path2)
```

Comment:
>Equals [](start = 24, length = 6)

This does not account for paths that are different only in case. 

`D:\GIT\LIBRARYMANAGER` and `d:\git\librarymanager` need to return equal on windows. 

Also this does not handle trailing `\` or `/`

`D:\git\LibraryManager\` and `D:\git\LibraryManager` should return equal.

Take a look at [RelativePathEqualityComparer](https://github.com/aspnet/LibraryManager/blob/master/src/LibraryManager/RelativePathEqualityComparer.cs)

Reply:
make related change here as this function does.

---
In reply to: [201064752](https://github.com/aspnet/LibraryManager/pull/286#discussion_r201064752) [](ancestors = 201064752)

Reply:
Can you file a separate issue on github to unify these two classes? 

---
In reply to: [201109962](https://github.com/aspnet/LibraryManager/pull/286#discussion_r201109962) [](ancestors = 201109962,201064752)

Reply:
Sure. Will do.

---
In reply to: [201131620](https://github.com/aspnet/LibraryManager/pull/286#discussion_r201131620) [](ancestors = 201131620,201109962,201064752)